Subject: [patch] remove whitespace after backslash
From: Robert Schwebel <r.schwebel@pengutronix.de>

With recent autotools I get this warning:
policy/Makefile.am:26: whitespace following trailing backslash

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 policy/Makefile.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: hal-0.5.12rc1/policy/Makefile.am
===================================================================
--- hal-0.5.12rc1.orig/policy/Makefile.am
+++ hal-0.5.12rc1/policy/Makefile.am
@@ -23,7 +23,7 @@ check:
 	      then \
                 echo ok; \
             else \
-                echo failed; \ 
+                echo failed; \
 		exit 1; \
             fi; \
         done;
